lib/pushmi_pullyu/swift_depositer.rb

```diff
@@ -23,27 +23,30 @@ def deposit_file(file_name, swift_container)
 
     era_container = swift_connection.container(swift_container)
 
-    deposited_file = if era_container.object_exists?(file_base_name)
-                       era_container.object(file_base_name)
-                     else
-                       era_container.create_object(file_base_name)
-                     end
+    object_exists = era_container.object_exists?(file_base_name)
 
     # Add swift metadata with in accordance to AIP spec:
     # https://docs.google.com/document/d/154BqhDPAdGW-I9enrqLpBYbhkF9exX9lV3kMaijuwPg/edit#
-    metadata = {
+    swift_metadata = {
       project: 'ERA',
       project_id: file_base_name,
       promise: 'bronze',
       aip_version: '1.0'
     }
 
     # ruby-openstack wants all keys of the metadata to be named like "X-Object-Meta-{{Key}}", so update them
-    metadata.transform_keys! { |key| "X-Object-Meta-#{key}" }
+    swift_metadata.transform_keys! { |key| "X-Object-Meta-#{key}" }
 
-    deposited_file.write(File.open(file_name),
-                         { 'etag' => checksum,
-                           'content-type' => 'application/x-tar' }.merge(metadata))
+    # creatre file metadata
```

Comment:
Majority of these little code comments aren't needed. Your code should be self explanatory. Telling me `era_container.create_object` creates a new file or creating a hash for file_metadata and telling me `create file metdata` is not needed as the code tells you this and is self explanatory.

Also you spelt `create` wrong here.